Source/WebCore/page/PointerLockController.h

```diff
@@ -45,7 +52,8 @@ class PointerLockController {
     WTF_MAKE_FAST_ALLOCATED;
 public:
     explicit PointerLockController(Page&);
-    void requestPointerLock(Element* target);
+    ~PointerLockController();
+    void requestPointerLock(Element* target, std::optional<PointerLockOptions>&&, RefPtr<DeferredPromise>);
```

Comment:
Seems we could have default parameter values for the 2 new parameters to simplify one of the call sites.